dart_canvas_parser/canvas_api.py

```diff
@@ -1,231 +1,96 @@
 import json
 import requests
-
 from django.conf import settings
-
 from canvas_sdk.exceptions import (CanvasAPIError, InvalidOAuthTokenError)
 from canvas_sdk.methods import courses, pages, quizzes, quiz_questions, \
     enrollments, users, modules
 from canvas_sdk.utils import get_all_list_data
 from canvas_sdk import RequestContext
-
 import logging
 
-log = logging.getLogger('dart')
 
+log = logging.getLogger('dart')
```

Comment:
I wonder if, given this is intended to be a separate library not necessarily tied to DART, we could review library best practices for (a) packaging, and (b) logging. This is fine for now though.